aten/src/ATen/native/native_functions.yaml

```diff
@@ -462,8 +462,10 @@
   device_check: NoCheck   # TensorIterator
   structured: True
   structured_inherits: TensorIteratorBase
+  ufunc_inner_loop:
+    Generic: add (AllAndComplex, BFloat16, Half)
+    ScalarOnly: add (Bool)
```

Comment:
Minor bikeshed comment: as someone casually looking at this, I find the `ScalarOnly` name to be confusing. In this context, it refers to non-vectorized, correct? Might be worth picking a different term to avoid clashing with `Scalar` used throughout the rest of `native_functions.yaml`.